docs/content/getting-started/install-traefik.md

```diff
@@ -0,0 +1,67 @@
+# Install Traefik
+
+The first step, before using all the awesome features of Traefik, is to install it.
+There are 3 main ways to install Traefik, you can:
```

Comment:
```suggestion
You can install Traefik with the following flavors:
```

Reply:
Ref: Simplifying (no need for "shiny" stuff here :) ) + not mentioning the number of bullet points